tests/tx/test_merged_mining.py

```diff
@@ -1403,6 +1403,9 @@ async def get_block_template(self, *, rules: List[str] = ['segwit'],
         await asyncio.sleep(self.response_delay)
         return stub
 
+    async def verify_block_proposal(self, *, block: bytes) -> Optional[str]:
+        return None
```

Comment:
TODO, right?

Reply:
Not really, I don't really want to port Bitcoin's verification algorithm. But it would be great if we had it. `None` means `valid`, the signature method could use a better docstring though.

Reply:
Ok! Is it just a mock for the tests?

Reply:
Yes.